README.md

```diff
@@ -1,3 +1,8 @@
 # The IBEX Developer's Manual
 
 [Click here to go to the GitHub wiki with the developer's manual.](https://isiscomputinggroup.github.io/ibex_developers_manual/)
+
+## Development workflow
+To run this manual under sphinx locally first create a virtual environment with `python -m venv .venv` and activate it. After this install the requirements by running `python -m pip install .`
+
+Then to build the sphinx documentation use `python -m sphinx-autobuild doc _build`.
```

Comment:
Should this just be a pointer to https://isiscomputinggroup.github.io/ibex_developers_manual/Editing-the-Wiki.html#building-the-wiki-locally ?

Reply:
yh m8